account_chart_update/wizard/wizard_chart_update.py

```diff
@@ -824,6 +835,57 @@ def _find_fiscal_positions(self, chart_template_ids, mapping_taxes,
                     'notes': notes,
                 })
 
+    @api.one
+    def _find_financial_reports(self):
+        wiz_fr = self.env['wizard.update.charts.accounts.financial.report']
+        # Remove previous financial reports
+        self.financial_report_ids.unlink()
+        # Search for new / updated accounts
+        root_account_id = self.chart_template_id.account_root_id.id
+        acc_templ_criteria = [
+            ('chart_template_id', '=', self.chart_template_id.id)]
+        if root_account_id:
+            acc_templ_criteria = ['|'] + acc_templ_criteria
+            acc_templ_criteria += [
+                '&', ('parent_id', 'child_of', [root_account_id]),
+                ('chart_template_id', '=', False)]
+        account_templates = self.env['account.account.template'].search(
+            acc_templ_criteria)
+        for account_template in account_templates:
+            template_fr_ids = set([fr.id for fr in
+                                   account_template.financial_report_ids])
+            # Ensure the account template is on the map (search for the mapped
+            # account id).
+            account = self.map_account_template(account_template)
+            if account:
```

Comment:
OK. I'm not a user of these reports, because in Spain we use https://github.com/OCA/l10n-spain/tree/8.0/account_balance_reporting, that allows us more flexibility and cross data from 2 years. We want to impulse it to a more general repo.